### DISCUSSION TITLE
How to update the Query Template for a number of verticals-either manually or by scope or ...?

### DISCUSSION THREAD

**Discussion by CoDiCor** [microsoft-search/pnp-modern-search#2425] [Q&A]
Hello, 
we configured a number of Search verticals that show dynamic search results based on a defined set of urls in the Query Template of the results web parts.
Example: {searchTerms} (path:"https://domain.com/sites1/" OR path:"https://domain.com/sites2/" )

The scope is 'LocalSharePointResults'

There are currently ca twenty urls in the Query Template and the list is growing.

What would be the best way to centrally update the Query Templates of the Search  Verticals?
Should I request a custom scope?

**Comment by wobba** [comment]
@CoDiCor If you can I would tie the sites to a hub site and filter on the hubid instead. Of if possible, have a URL prefix scheme so you can filter on https://domain.com/prefix*

**Reply by CoDiCor** [reply]
Thank  you very much for the advice! This might work for us.
